### PR TITLE
Add connections ontology setup w/out S2 specifics

### PR DESCRIPTION
**Pull request by Ray725**

### CODE REVIEW

**Comment by charbull** [comment]
@tasodorff friendly ping :)

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml [hunk 1]
```diff
@@ -28,6 +28,12 @@ VAV:
   - EQUIPMENT
   opt_uses:
   - zone_use_label # Needed until the zones can are joined to assets with connections.
+  connections:
+    ROOM: CONTAINS
+    AHU: FEEDS
```
Comment (charbull) [inline ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:33]: can you please update with the cardinality and this new form


```
 connections:
 AHU:
   type: FEEDS
   min: 
   max:
```

refer to the values from the sheets doc

Comment (berkoben) [reply ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:33]: update to this: 
 AHU:
   type: FEEDS
   min: <number>
   max: <number>
   exact_type: <true/false>

The simple form (current) is also valid.  It is equivalent to:

AHU:
  type: FEEDS
  min: 1
  max: 1
  exact_type: false

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml [hunk 2]
```diff
@@ -28,6 +28,12 @@ VAV:
   - EQUIPMENT
   opt_uses:
   - zone_use_label # Needed until the zones can are joined to assets with connections.
+  connections:
```
Comment (berkoben) [inline ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:31]: this is not true for all equipment.

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml (hunk 1 above)
Comment (Ray725) [reply ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:33]: Done

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml (hunk 2 above)
Comment (Ray725) [reply ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:31]: Okay, assuming this is in reference to `zone_use_label`, deleting

**Pull request closed without merging**